### DISCUSSION TITLE
Ressources limits

### DISCUSSION THREAD

**Discussion by Nonolanlan1007** [coollabsio/coolify#2636] [Q&A] [answered]
Hi!
Does Ressources limits apply to deployment builds?

**Comment by peaklabs-dev** [comment]
@Nonolanlan1007 What are you deploying?

**Reply by Nonolanlan1007** [reply]
A Rust app with Nixt Packs

**Reply by peaklabs-dev** [reply, marked as answer]
@Nonolanlan1007 With Nixpacks you should have access to Resource Limits in the configuration. For other things, Resource Limits is not yet added.

**Reply by Nonolanlan1007** [reply]
Ok thanks 👌
